<ui version="4.0" >
 <class>lp_ladspa_lister</class>
 <widget class="QWidget" name="lp_ladspa_lister" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>393</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="2" column="1" colspan="3" >
    <widget class="QLabel" name="lb_copyright" >
     <property name="minimumSize" >
      <size>
       <width>300</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text" >
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3" >
    <widget class="QLabel" name="lb_maker" >
     <property name="minimumSize" >
      <size>
       <width>300</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text" >
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3" >
    <widget class="QLabel" name="lb_plugname" >
     <property name="minimumSize" >
      <size>
       <width>300</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text" >
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item rowspan="4" row="0" column="0" >
    <widget class="QTreeView" name="ls_plugins" >
     <property name="minimumSize" >
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>600</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="3" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>111</width>
       <height>141</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="3" >
    <widget class="QPushButton" name="pb_ok" >
     <property name="text" >
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" >
    <widget class="QPushButton" name="pb_cancel" >
     <property name="text" >
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>281</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
